fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NamespaceTools.java

```diff
@@ -75,7 +76,13 @@ public static void validatePath(final Session session, final String path) {
                     try {
                         namespaceRegistry.getURI(prefix);
                     } catch (final NamespaceException e) {
-                        throw new FedoraInvalidNamespaceException("Prefix " + prefix + " has not been registered", e);
+                        if (segment.equals("fcr:backup")) {
```

Comment:
Let us say that he did not raise it. I don't know whether he thought about it, but he can comment if he did. Please do take a look at putting this in `FedoraExport`, thank you.